win/CS/HandBrakeWPF/ViewModels/SubtitlesViewModel.cs

```diff
@@ -553,6 +588,43 @@ public void SetSource(Source source, Title title, Preset preset, EncodeTask task
             this.AutomaticSubtitleSelection();
         }
 
+        /// <summary>
+        /// Checks the configuration of the subtitles and warns the user about any potential issues.
+        /// </summary>
+        public bool ValidateSubtitles()
+        {
+            var nonBurnedSubtitles = this.Task.SubtitleTracks.Where(subtitleTrack => !subtitleTrack.Burned).ToList();

```

Comment:
If I recall correctly there is a libhb functions that determine how tracks can be added. So typically the UI simply won't allow tracks to be added where they are incompatible and there is already a burn-in track.

Reply:
This method was mostly informed by GTK's validation logic. If there are functions in libhb, they're not used presently by any GUI.

https://github.com/HandBrake/HandBrake/blob/1728150b7e088eccc89f03ad03f9ea9dcb26cade/gtk/src/hb-backend.c#L4324-L4395

Reply:
Without this validation, the GUI iirc will allow multiple tracks however they'd be ignored by libhb upon processing. The warning is to give the user a chance to adjust their expectations prior to encoding.

Reply:
*blinks* I could have sworn I had written full validation on this tab to prevent this but your right, it doesn't.  Maybe I have an old branch kicking around somewhere :(

I'll probably do some digging and do some refactoring of this section of code in post so it can handle numerous situations that can occur beyond WebM.

You can ignore my comment for now then :)